app/src/main/res/values/strings.xml

```diff
@@ -576,8 +576,8 @@
         <item>144p</item>
     </string-array>
     <!-- Updates Settings -->
-    <string name="updates_setting_title">Updates</string>
-    <string name="updates_setting_description">Show a notification to prompt app update when a new version is available</string>
+    <string name="updates_setting_title">Upgrades</string>
+    <string name="updates_setting_description">Ask to install new versions</string>
```

Comment:
This has a completly different meaning

Reply:
Then I have misunderstood what this means in the first place.
I thought it was about installing new versions of NewPipe

Reply:
```suggestion
    <string name="updates_setting_description">Ask to install new versions of NewPipe</string>
```